.changeset/spotty-falcons-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`Math`, `SafeMath`: Replace simple ternary operations `cond ? a : b` by a more efficient branchless `ternary` function.
```

Comment:
Just came to the conclusion that the optimization itself is not a substantial change. We should focus the changelog entry on the new `ternary` function and the reasons why someone would use it.

```suggestion
`Math`, `SignedMath`: Add a branchless `ternary` function that computes`cond ? a : b` in constant gas costs.
```